src/helm/benchmark/run_specs/bhasa_run_specs.py

```diff
@@ -0,0 +1,678 @@
+from helm.benchmark.adaptation.adapter_spec import (
+    ADAPT_MULTIPLE_CHOICE_SEPARATE_ORIGINAL,
+)
+from helm.benchmark.adaptation.common_adapter_specs import (
+    get_generation_adapter_spec,
+    get_multiple_choice_separate_adapter_spec,
+)
+from helm.benchmark.metrics.bhasa_metrics import (
+    get_bhasa_summarization_metric_specs,
+    get_bhasa_machine_translation_metric_specs,
+    get_bhasa_qa_metric_specs,
+)
+from helm.benchmark.metrics.common_metric_specs import (
+    get_basic_metric_specs,
+    get_classification_metric_specs,
+    get_exact_match_metric_specs,
+    get_f1_metric_specs,
+)
+from helm.benchmark.run_spec import RunSpec, run_spec_function
+from helm.benchmark.scenarios.scenario import ScenarioSpec
+
+# BHASA Run Specs
+#   A. Natural Language Understanding
+#   B. Natural Language Generation
+#   C. Natural Language Reasoning
+#   D. Linguistic Diagnostics
+
+# A. Natural Language Understanding
+#   1. Question Answering
+#   2. Sentiment Analysis
+#   3. Toxicity Detection/Classification
+
+# 1. Question Answering
+# 1.1 Indonesian: TyDiQA
+@run_spec_function("tydiqa")
+def get_tydiqa_spec() -> RunSpec:
+    name = "tydiqa"
+
+    adapter_spec = get_generation_adapter_spec(
+        instructions="Anda akan diberikan sebuah paragraf dan sebuah pertanyaan. Jawablah pertanyaannya dengan mengekstrak jawaban dari paragraf tersebut.",
+        output_noun="Jawaban",
+        stop_sequences=["\n"],
+        max_tokens=256,
+    )
+
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.bhasa_scenario.TyDiQAScenario"
+    )
+
+    return RunSpec(
+        name=name,
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_bhasa_qa_metric_specs(args={
+            "language": "id",
+        }),
+        groups=["bhasa_nlu"],
```

Comment:
For these and all other specs: do `groups=["tydiqa", "bhasa_nlu"]` i.e. also have the scenario itself as a group name.

Reply:
Fixed!